Jenkinsfile

```diff
@@ -59,34 +113,58 @@ pipeline {
-          }
+					withCredentials([file(credentialsId: 'Config', variable: 'ConfigFile')]) {
+						sh "java -jar target/biomodels-*-jar-with-dependencies.jar $ConfigFile ${env.ABS_RELEASE_PATH}/biomodels/models2pathways.tsv"
+					}
 				}
 			}
 		}
 		stage('Post: Back up DB'){
 			steps{
 				script{
-          withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-            def release_current_after_biomodels_dump = "${env.RELEASE_CURRENT}_${currentRelease}_after_biomodels.dump"
-            sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > $release_current_after_biomodels_dump"
-            sh "gzip -f $release_current_after_biomodels_dump"
-          }
+					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+						def release_current_after_biomodels_dump = "${env.RELEASE_CURRENT}_${currentRelease}_after_biomodels.dump"
+						sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > $release_current_after_biomodels_dump"
+						sh "gzip -f $release_current_after_biomodels_dump"
+					}
 				}
 			}
 		}
-		stage('Post: Archive logs and backups'){
+		stage('Post: Archive Outputs'){
 			steps{
 				script{
-          sh "mkdir -p archive/${currentRelease}/logs"
-          sh "mv --backup=numbered *_${currentRelease}_*.dump.gz archive/${currentRelease}/"
-          sh "gzip logs/*"
-          sh "mv logs/* archive/${currentRelease}/logs/"
+					def s3Path = "${env.S3_RELEASE_DIRECTORY_URL}/${currentRelease}/biomodels"
+					def biomodelsPath = "${env.ABS_RELEASE_PATH}/biomodels"
+					sh "mkdir -p databases/ data/"
+					sh "mv --backup=numbered *_${currentRelease}_*.dump.gz databases/"
+					sh "mv graph-importer/logs/* logs/"
+					sh "mv analysis-core/logs/* logs/"
+					sh "mv ${biomodelsPath}/logs/* logs/"
+					sh "mv ${biomodelsPath}/jsbml.log logs/"
+					sh "mv ${biomodelsPath}/models2pathways.tsv data/"
+					sh "mv analysis-core/analysis-biomodels-v${currentRelease}.bin data/"
+					sh "mv /tmp/intact-micluster.txt data/"
+					sh "gzip data/* logs/*"
+					sh "mv graph-importer/graphdb_${currentRelease}_biomodels.tgz data/"
+					sh "mv ${biomodelsPath}/BioModels_Database-*sbml_files.tar.bz2 data/"
+					sh "aws s3 --no-progress --recursive cp databases/ $s3Path/databases/"
+					sh "aws s3 --no-progress --recursive cp logs/ $s3Path/logs/"
+					sh "aws s3 --no-progress --recursive cp data/ $s3Path/data/"
+					sh "rm -r databases logs data ${biomodelsPath}/BioModels_Database-*-sbml_files"
+					sh "rm -rf graph-importer*"
+					sh "rm -rf analysis-core*"
+					sh "rm -rf release-jenkins-utils*"
 				}
 			}
 		}
 	}		
 }
-		
-		
+// Utility function that checks if a git directory exists. If not, it is cloned.
+def cloneOrPullGitRepo(String repoName) {
```

Comment:
Similar to https://github.com/reactome/release-jenkins-utils/pull/3#discussion_r436201736 - this function will be useful across the Jenkins Release Pipeline.  If it can be moved to a new utilitiy file that is agnostic of any particular release step and imported where needed, it would reduce the duplication across all the Jenkins steps/modules.